recipe/bld.bat

```diff
@@ -13,7 +13,8 @@ cmake .. -GNinja                                 ^
   -DAVIF_CODEC_AOM=ON                            ^
   -DAVIF_CODEC_SVT=OFF                           ^
   -DAVIF_CODEC_DAV1D=ON                          ^
-  -DAVIF_CODEC_LIBGAV1=OFF
+  -DAVIF_CODEC_LIBGAV1=OFF                       ^
+  -DAVIF_ENABLE_WERROR=OFF
```

Comment:
out of curiosity what kind of warnings you have ? as CF does not have this option enabled

Reply:
These two warnings were making the build fail on `win`:

```
(C:\Users\Lorenzo\miniconda3\conda-bld\debug_1689085451863\_build_env) C:\Users\Lorenzo\miniconda3\conda-bld\debug_1689085451863\work\build>ninja
[1/11] Building C object CMakeFiles\avif.dir\src\exif.c.obj
FAILED: CMakeFiles/avif.dir/src/exif.c.obj
C:\PROGRA~2\MICROS~2\2017\COMMUN~1\VC\Tools\MSVC\1410~1.250\bin\HostX86\x86\cl.exe  /nologo -DAVIF_BUILDING_SHARED_LIBS -DAVIF_CODEC_AOM=1 -DAVIF_CODEC_AOM_DECODE=1 -DAVIF_CODEC_AOM_ENCODE=1 -DAVIF_CODEC_DAV1D=1 -DAVIF_DLL -Davif_EXPORTS -IC:\Users\Lorenzo\miniconda3\conda-bld\debug_1689085451863\work\include -IC:\Users\Lorenzo\miniconda3\conda-bld\debug_1689085451863\_h_env\Library\include /DWIN32 /D_WINDOWS /MD /O2 /Ob2 /DNDEBUG /W4 /wd4324 /wd4996 /source-charset:utf-8 /execution-charset:us-ascii /WX /showIncludes /FoCMakeFiles\avif.dir\src\exif.c.obj /FdCMakeFiles\avif.dir\ /FS -c C:\Users\Lorenzo\miniconda3\conda-bld\debug_1689085451863\work\src\exif.c
C:\Users\Lorenzo\miniconda3\conda-bld\debug_1689085451863\work\src\exif.c(32): error C2220: warning treated as error - no 'object' file generated
C:\Users\Lorenzo\miniconda3\conda-bld\debug_1689085451863\work\src\exif.c(32): warning C4204: nonstandard extension used: non-constant aggregate initializer
[4/11] Building C object CMakeFiles\avif.dir\src\read.c.obj
FAILED: CMakeFiles/avif.dir/src/read.c.obj
C:\PROGRA~2\MICROS~2\2017\COMMUN~1\VC\Tools\MSVC\1410~1.250\bin\HostX86\x86\cl.exe  /nologo -DAVIF_BUILDING_SHARED_LIBS -DAVIF_CODEC_AOM=1 -DAVIF_CODEC_AOM_DECODE=1 -DAVIF_CODEC_AOM_ENCODE=1 -DAVIF_CODEC_DAV1D=1 -DAVIF_DLL -Davif_EXPORTS -IC:\Users\Lorenzo\miniconda3\conda-bld\debug_1689085451863\work\include -IC:\Users\Lorenzo\miniconda3\conda-bld\debug_1689085451863\_h_env\Library\include /DWIN32 /D_WINDOWS /MD /O2 /Ob2 /DNDEBUG /W4 /wd4324 /wd4996 /source-charset:utf-8 /execution-charset:us-ascii /WX /showIncludes /FoCMakeFiles\avif.dir\src\read.c.obj /FdCMakeFiles\avif.dir\ /FS -c C:\Users\Lorenzo\miniconda3\conda-bld\debug_1689085451863\work\src\read.c
C:\Users\Lorenzo\miniconda3\conda-bld\debug_1689085451863\work\src\read.c(948): error C2220: warning treated as error - no 'object' file generated
C:\Users\Lorenzo\miniconda3\conda-bld\debug_1689085451863\work\src\read.c(948): warning C4244: '-=': conversion from 'uint64_t' to '::size_t', possible loss of data
[6/11] Building C object CMakeFiles\avif.dir\src\stream.c.obj
ninja: build stopped: subcommand failed.
```